python/hail/plot/plots.py

```diff
@@ -146,21 +147,36 @@ def scatter(x, y, label=None, title=None, xlabel=None, ylabel=None, size=4, lege
         Size of markers in screen space units.
     legend : bool
         Whether or not to show the legend in the resulting figure.
+    collect_all : bool
```

Comment:
You might remark this is ignored if the input is a python object (e.g. it was already collected!)